Kitodo/pom.xml

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.kitodo.production</groupId>
+    <artifactId>kitodo-produktion</artifactId>
```

Comment:
Please use English naming (“kitodo-production” with “c”, not “k”). When checking out the project with Eclipse, a folder named like this string is generated.